docs/countme.md

```diff
@@ -25,17 +25,25 @@ standalone way.
 
 ## Disabling DNF Count Me on a system
 
-To disable this feature, you need to update the Count Me option in all RPM
-repository configuration files:
+To disable this feature, you need to stop and disable the
+`rpm-ostree-countme.timer` and mask the corresponding units as a precaution:
 
 ```
-sed -i 's/countme=1/countme=0/g' /etc/yum.repos.d/*.repo
+$ systemctl disable --now rpm-ostree-countme.timer
+$ systemctl mask rpm-ostree-countme.service rpm-ostree-countme.timer
```

Comment:
I'd condense this to `systemctl mask --now rpm-ostree-countme.{service,timer}`.

Reply:
Really just disabling/masking the timer should be enough right?

Also actually related to this I thought we had some discussion about pulling in countme by default via `Wants=` - did that happen somewhere?  And if so, we should `Wants=rpm-ostree-countme.timer` right?

Reply:
OK right the `Wants=` is in https://src.fedoraproject.org/rpms/fedora-release/pull-request/173#request_diff

So I think just `systemctl mask --now rpm-ostree-countme.timer` is enough.  The service shouldn't be started outside of that.

Reply:
I wanted to be completely exhaustive but I guess that makes it harder / scarier than necessary.